flocker/control/httpapi.py

```diff
@@ -568,14 +606,24 @@ def container_configuration_response(application, node):
     """
     result = {
         "host": node, "name": application.name,
-        "image": application.image.full_name
+        "image": application.image.full_name,
+        "restart_policy": {
```

Comment:
If restart policy is *always* included you should update examples appropriately. Alternatively, you could omit restart policy if it's the default of never.